src/components/LoginForm.tsx

```diff
@@ -37,36 +29,48 @@ const Login = (): JSX.Element => {
       navigate('/patients')
     } catch (error) {
       console.error(error)
-      if (axios.isAxiosError(error) && (error.response?.status === 400)) {
-        addErrorCallback({ body: 'Invalid credentials' })
-      } else {
-        addErrorCallback({ body: 'Unknown login error' })
-      }
+      setError(t('error.errorLogin'))
     }
   }
 
   const onFormSubmit = (e: React.FormEvent): void => e.preventDefault()
 
   return (
-    <Form className='d-flex' onSubmit={onFormSubmit}>
-      <Form.Control
-        placeholder='Username'
-        className='me-2'
-        aria-label='Email'
-        onChange={onUsernameChange}
-      />
-      <Form.Control
-        type='password'
-        placeholder='Password'
-        className='me-2'
-        aria-label='Password'
-        onChange={onPasswordChange}
-      />
-      <Button type='submit' onClick={onLogin}>
+    <Form onSubmit={onFormSubmit}>
+      <Form.Group className='mb-3'>
+        <Form.Label>{t('form.userName')}</Form.Label>
+        <Form.Control
+          required
+          type='text'
+          onChange={onUsernameChange}
+          isInvalid={Boolean(error)}
+        />
+        {error !== '' && (
+          <Form.Control.Feedback type='invalid'>
+            {error}
+          </Form.Control.Feedback>
+        )}
+      </Form.Group>
+
+      <Form.Group className='mb-3'>
+        <Form.Label>{t('form.password')}</Form.Label>
+        <Form.Control
+          required
+          type='password'
+          onChange={onPasswordChange}
+          isInvalid={Boolean(error)}
+        />
+        {error !== '' && (
+          <Form.Control.Feedback type='invalid'>
+            {error}
+          </Form.Control.Feedback>
```

Comment:
If it's the same global error, we may not want to repeat it for each fields, it could look weird & ugly, what do you think?